react/react-addons-test-utils.d.ts

```diff
@@ -6,7 +6,7 @@
 /// <reference path="react.d.ts" />
 
 declare namespace __React {
-    interface SyntheticEventData {
+    interface SyntheticEventData extends Event {
```

Comment:
Yeah, that's an option. At least in the current version of TypeScript that's probably the best we can do.